src/components/map/Map.tsx

```diff
@@ -191,6 +193,7 @@ export const Map = ({
         )}
         {locations?.map((marker, index) => {
           const isActiveMarker = selectedMarker && marker.id === selectedMarker;
+          const calloutText = marker.title || undefined;
```

Comment:
`|| undefined` is not necessary. `marker.title` returns `undefined` if it is not defined.